test/models/genre_test.rb

```diff
@@ -16,4 +16,31 @@ class GenreTest < ActiveSupport::TestCase
     assert_not genre.normalized_name.nil?
     assert_equal 'iouaaa', genre.normalized_name
   end
+
+  test 'should be able to merge genres' do
+    genre1 = create(:genre)
+    genre2 = create(:genre)
+
+    assert genre2.merge(genre1)
```

Comment:
If the assumption is that `merge` shouldn't fail, `assert`ing it's return value doesn't make much sense.